catalogue_pipeline/terraform/security_groups.tf

```diff
@@ -1,7 +1,7 @@
 resource "aws_security_group" "service_egress_security_group" {
-  name        = "${var.namespace}_service_egress_security_group"
+  name        = "pipeline_service_egress_security_group"
```

Comment:
Why taking this sg out of the pipeline module? I don't see it ever being used outside

Reply:
Oh yeah, I hadn't spotted we only need half of this file. I’ll roll it back.